cmd/coder/main.go

```diff
@@ -15,6 +20,48 @@ import (
 func main() {
 	rand.Seed(time.Now().UnixMicro())
 
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
```

Comment:
We might want to extract and place in `enterprise/cmd/coder` as well.

Reply:
I've pulled out the function into AGPL `cli` dir and added it solely to `server` and `agent` (which are both called by their enterprise counterparts) so we should be all good